.test-infra/jenkins/job_Dependency_Check.groovy

```diff
@@ -1,73 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import CommonJobProperties as commonJobProperties
-
-job('beam_Dependency_Check') {
-  description('Runs Beam dependency check.')
-
-  // Set common parameters.
-  commonJobProperties.setTopLevelMainJobProperties(
-      delegate, 'master', 100, true, 'beam', false)
-
-  // Allows triggering this build against pull requests.
-  commonJobProperties.enablePhraseTriggeringFromPullRequest(
-      delegate,
-      'Beam Dependency Check',
-      'Run Dependency Check',
-      false
-      )
-
-  commonJobProperties.setAutoJob(
-      delegate,
-      '@weekly')
-
-  steps {
-    gradle {
-      rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks('runBeamDependencyCheck')
```

Comment:
Actually, I think we need to rip out this piece too.

This has a gradle task https://github.com/apache/beam/blob/4f07cda050cd144a95a686c4138307e9920f313d/build.gradle.kts#L583 which then calls into the python sdk dependency check code - https://github.com/apache/beam/blob/4f07cda050cd144a95a686c4138307e9920f313d/sdks/python/scripts/run_dependency_check.sh#L22

There's some more references that will need cleaned up around this, https://github.com/search?q=repo%3Aapache%2Fbeam%20dependencyUpdates&type=code is a good starting point

Reply:
Sorry, missed that initially

Reply:
I think I knocked them all out by checking `dependecyCheck`, `dependency-check`, and `dependencyUpdate` references along with the two direct references to the shell script itself. Fortunately touching things within the `sdks/` folder triggers more of the automated testing so there will at least be an early signal of if this really breaks something